ops/terraform/remote_files/scripts/install-node.sh

```diff
@@ -29,6 +29,26 @@ function install-docker() {
   sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
 }
 
+function install-git() {
+  echo "Installing Git..."
+  sudo apt-get update -y
+  sudo apt-get install -y git
+  echo "...Git installation is complete!"
+}
+
+function install-git-lfs() {
+  echo "Installing Git Large File Storage (LFS)..."
+  # Update package lists
+  sudo apt-get update
+  # Add git-lfs package repository
+  curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
+  # Install git-lfs
+  sudo apt-get install -y git-lfs
```

Comment:
are you sure you don't want to pin a specific version here?

Reply:
No, they don't specify any version and as we as we are just goin to use the clone functionality